### ISSUE TITLE
fetch_trades() on ZB results in Exception: 'bool' object is not subscriptable

### ISSUE THREAD

**Issue by JoshVarty** [ccxt/ccxt#3323]
- OS: Windows/Ubuntu
- Programming Language version: `3.6.3`
- CCXT version: `1.15.7`
- Exchange: ZB
- Method: `fetch_trades()`

Certain pairs seem to trigger this. `load_markets()` suggests that the pairs exist on the exchange, but the exchange returns some kind of error message which seems to cause CCXT to throw an error.

```Python
import ccxt

exchange = getattr(ccxt, "zb")()
markets = exchange.load_markets()

print('ICX/BTC' in markets.keys())		//True

exchange.verbose = True
pair = 'ICX/BTC'
trades = exchange.fetch_trades(pair)
```

Output
```
True

Request: GET http://api.zb.com/data/v1/trades?market=icx_btc {'User-Agent': 'python-requests/2.19.1', 'Accept-Encoding': 'gzip, deflate'} None

Response: GET http://api.zb.com/data/v1/trades?market=icx_btc 200 {'Server': 'nginx', 'Date': 'Wed, 04 Jul 2018 19:56:53 GMT', 'Content-Type': 'text/javascript;charset=utf-8', 'Content-Length': '44', 'Connection': 'keep-alive', 'Access-Control-Allow-Origin': '*', 'CNDServer': 'cache_server_1'} {"result":false,"message":"服务端忙碌"}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Program Files\Python36\lib\site-packages\ccxt\zb.py", line 335, in fetch_trades
    return self.parse_trades(response, market, since, limit)
  File "C:\Program Files\Python36\lib\site-packages\ccxt\base\exchange.py", line 1190, in parse_trades
    array = [self.parse_trade(trade, market) for trade in array]
  File "C:\Program Files\Python36\lib\site-packages\ccxt\base\exchange.py", line 1190, in <listcomp>
    array = [self.parse_trade(trade, market) for trade in array]
  File "C:\Program Files\Python36\lib\site-packages\ccxt\zb.py", line 314, in parse_trade
    timestamp = trade['date'] * 1000
TypeError: 'bool' object is not subscriptable
```

**Comment by JoshVarty** [comment]
Google translate says that the message `服务端忙碌` translates to `Busy Server`. However, only certain pairs seem to trigger this message. 

Either way, perhaps there is a better way to fail or throw a clearer exception?

**Comment by kroitor** [comment]
Fixed in 1.15.10. Thx!
